src/main/resources/application-test.properties

```diff
@@ -0,0 +1,13 @@
+data.file.path=src/main/resources/dataToFillMaps.json
+
+# jdbc.X
+jdbc.driverClassName=org.postgresql.Driver
+jdbc.url=jdbc:postgresql://localhost:5432/gym
+jdbc.user=armine
+jdbc.pass=0000
+
+# hibernate.X
+hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
We could use in-memory db like h2 and hsqldb for test purposes. Not real postgres instance

Reply:
fixed